### ISSUE TITLE
'oidc_state' does not match the OIDC callback state

### ISSUE THREAD

**Issue by rackymuthu** [mozilla/mozilla-django-oidc#303]
I'm run the django project using the Mozilla Django OIDC. While the project running it is shown the **'oidc_state' does not match the OIDC callback state** error rise in console. how to fix

**Comment by johngian** [comment]
Hey @rackymuthu. Can you provide some more information about your dev environment ?

**Comment by stevejalim** [comment]
I'm seeing this intermittently, too, on a Wagtail project using `mozilla-django-oidc`. 

Sometimes, when opening a new page/following a link in the Wagtail CMS UI, it'll 500.

I don't currently have a full traceback, but I screenshotted the top-line details when I managed ti trigger it in local dev. (Fairly usless, sorry.) 

![Screenshot 2019-11-01 at 13 40 53](https://user-images.githubusercontent.com/101457/68115403-3cf7f500-fef0-11e9-8a80-6e99ba6e402f.png)

The project is using the middleware that re-confirms the user's authorisation with the OpenID provider, at the default 15-minute interval. My gut says this _might_ be a factor, but I haven't yet looked at anything proper. 

Will update this ticket if/when I get a full traceback.
